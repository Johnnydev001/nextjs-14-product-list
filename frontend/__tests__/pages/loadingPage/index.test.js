import {render} from "@testing-library/react";
import LoadingProducts from "../../../src/app/loading";

describe('Loading products page tests', () => {
    it('should render the loading products page and match snapshot', () => {
        const {container, getByText } = render(<LoadingProducts/>)

        expect(getByText('Loading products...')).toBeInTheDocument()
        expect(container).toMatchSnapshot()
    })
})