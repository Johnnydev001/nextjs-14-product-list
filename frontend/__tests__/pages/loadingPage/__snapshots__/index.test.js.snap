// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading page tests should render the loading page 1`] = `
<div>
  <div
    aria-live="assertive"
    class="text-center flex justify-center fixed top-0 right-0  left-0 z-30 bg-zinc-100 text-black px-4 py-2 rounded-md text-md"
    role="alert"
  >
    Loading products...
  </div>
</div>
`;

exports[`Loading products page tests should render the loading products page 1`] = `
<div>
  <div
    aria-live="assertive"
    class="text-center flex justify-center fixed top-0 right-0  left-0 z-30 bg-zinc-100 text-black px-4 py-2 rounded-md text-md"
    role="alert"
  >
    Loading products...
  </div>
</div>
`;

exports[`Loading products page tests should render the loading products page and match snapshot 1`] = `
<div>
  <div
    aria-live="assertive"
    class="text-center flex justify-center fixed top-0 right-0  left-0 z-30 bg-zinc-100 text-black px-4 py-2 rounded-md text-md"
    role="alert"
  >
    Loading products...
  </div>
</div>
`;
